assets/src/components/app.tsx

```diff
@@ -34,4 +38,28 @@ const App = (): JSX.Element => {
   )
 }
 
+export const useFetchRoutes = (dispatch: Dispatch): void => {
```

Comment:
What do you think about moving these each to its own file in `hoooks/` or something like that? I worry that this App component file might get more and more sprawling as we continue to add more of these. As long as the hooks are named well (which I think these are) I think we can understand the behavior where they are used while keeping each file more focused.